sub-pages/_single.html

```diff
@@ -95,4 +98,7 @@ <h2 class="h3">
     </aside>
     {% endif %}
 
+    {% set office = vars.office %}
```

Comment:
So that is being used by the office-contact.html. Might be able to set it in the includes file.